swagger.yaml

```diff
@@ -211,7 +211,44 @@ paths:
             $ref: '#/definitions/User'
         '401':
           $ref: '#/responses/401'
-
+  '/users':
+    x-swagger-router-controller: mock
+    get:
+      description: |
+        Request collection of Registered Users
+      produces:
+        - application/json
+      parameters:
+        - name: type
+          description: |
+            Filter users by user type.
+            Possible values are user, moderator, employee, agent, admin and expert
+          required: false
+          in: query
+          type: array
+          items:
+            type: string
```

Comment:
you could potentailly make this an enum
https://swagger.io/docs/specification/data-models/enums/

but it might be a problem because it's an array of any length of these, I believe to remember this being harder to do

Reply:
Do you mean under items? Like 
```
       in: query
          type: array
          items:
            type: string
            enum:
              - user
              - moderator
              - employee
              - agent
              - admin
              - expert
``` 
?

Reply:
yes, assuming this can be parsed like this